src/sentry/models/release_threshold/constants.py

```diff
@@ -41,32 +41,24 @@ def as_str_choices(cls):
 
 
 class TriggerType:
-    PERCENT_OVER = 0
-    PERCENT_UNDER = 1
-    ABSOLUTE_OVER = 2
-    ABSOLUTE_UNDER = 3
+    OVER = 0
```

Comment:
We don't need % over/under
The threshold type will either be a count or a % itself.

Reply:
where are you defining whether it's a count or a %?

Reply:
This will be in the threshold type (as opposed to the trigger type)

So "Crash Free Rate"s will always be a %age
Error Counts will always be a number

we'll never have a % type with an 'absolute over' trigger type